src/Microsoft.ML.Tokenizers/Model/BPETokenizer.cs

```diff
@@ -226,6 +287,9 @@ public static async Task<BpeTokenizer> CreateAsync(
         /// <param name="continuingSubwordPrefix">The prefix to attach to sub-word units that don’t represent a beginning of word.</param>
         /// <param name="endOfWordSuffix">The suffix to attach to sub-word units that represent an end of word.</param>
         /// <param name="fuseUnknownTokens">Indicate whether allowing multiple unknown tokens get fused.</param>
+        /// <param name="byteLevel">Indicate whether to handle the input text in byte level.</param>
+        /// <param name="beginningOfSentenceToken">The beginning of sentence token.</param>
+        /// <param name="endOfSentenceToken">The end of sentence token.</param>
         private BpeTokenizer(
```

Comment:
Would it be a good idea to have a constructor here that takes a `BpeOptions` so you don't have to change the signature everytime you have something like this?

Reply:
This will require to have the existing `BpeTokenizer.Create(..., ..., ...,..)` to always create the options object to call that constructor. It may not be a big deal to do so but it will need some work to read the vocab and merge files into IEnumerable to set it to the options. Otherwise, we could do it but duplicate some code between the two constructors.